src/test/java/io/nats/client/api/StreamConfigurationTests.java

```diff
@@ -107,19 +107,23 @@ public void testConstruction() {
         assertEquals(sources.get(0).getExternal(), external);
         assertEquals(sources.get(0).getExternal().hashCode(), external.hashCode());
 
-        List<String> lines = ResourceUtils.dataAsLines("SourceBaseJson.txt");
+        List<String> lines = ResourceUtils.dataAsLines("MirrorsSources.json");
         for (String l1 : lines) {
-            Mirror m1 = new Mirror(l1);
-            assertEquals(m1, m1);
-            assertNotEquals(m1, null);
-            assertNotEquals(m1, new Object());
-            for (String l2 : lines) {
-                Mirror m2 = new Mirror(l2);
-                if (l1.equals(l2)) {
-                    assertEquals(m1, m2);
-                }
-                else {
-                    assertNotEquals(m1, m2);
+            if (l1.startsWith("{")) {
+                Mirror m1 = new Mirror(l1);
+                assertEquals(m1, m1);
+                assertNotEquals(m1, null);
```

Comment:
```suggestion
                assertNotNull(m1);
```

Reply:
this is specifically exercising the equals testing m1 versus null. I have changed it to this to be clear
```
//this provides testing coverage 
//noinspection ConstantConditions,SimplifiableAssertion
assertTrue(!m1.equals(null));
```